### DISCUSSION TITLE
Creating a Documentation team

### DISCUSSION THREAD

**Discussion by kjaymiller** [BlackPythonDevs/blackpythondevs#63] [Programs]
# Create Documentation Team

## Target Launch Date: 

1 June 2024

## Goal: 

Create a group lead by a BPD Council Leader. They are responsible for updating documenation across the website as needed.

## Description:

We want to create a group for folks that are wanting to get into Documentation. This would be our documentation group and would focus on giving folks the ability to contribute to the community while learning valuable documentation coordination skills around open source.

### Current

Working with documentation professionals for skilling opportunities

### Future

Financial assistance for documentation events

---

tagged issue: #62

**Comment by TimOsahenru** [comment]
interested @kjaymiller

**Comment by activus-d** [comment]
@kjaymiller I would be happy to help with documentation

**Comment by HectorGitt** [comment]
Interested

**Comment by SilverWhisk** [comment]
Interested @kjaymiller

**Comment by Barderus** [comment]
I'm interested as well! @kjaymiller

**Comment by KaneahAbagale** [comment]
I'm interested

**Comment by AtienoObwanda** [comment]
Hello @kjaymiller, I am interested!

**Comment by kjaymiller** [comment]
Thank you so much for your interest. We will temporarily lock this and I will work with folks in the discord. Please check out the #Documentation-Team Channel
